charts/spinnaker/values.yaml

```diff
@@ -170,43 +170,36 @@ kubeConfig:
   # Use this when you want to register arbitrary clusters with Spinnaker
   # Upload your ~/kube/.config to a secret
   enabled: false
-  # Creates a Spinnaker account per context in `.kubeConfig.contexts` with the
-  # same name. Can be used together with
-  # `.kubeConfig.accountPerContextAndNamespace`.
-  accountPerContext: true
-  # Creates a Spinnaker account for each combination of context and namespace
-  # as per `contexts` and `namespacePartitionedAccounts`. Can be used together
-  # with`accountPerContext`.
-  accountPerContextAndNamespace: false
   # secretName: my-kubeconfig
   # secretKey: config
   # Use this when you want to configure halyard to reference a kubeconfig from s3
   # This allows you to keep your kubeconfig in an encrypted s3 bucket
   # For more info visit:
   #   https://www.spinnaker.io/reference/halyard/secrets/s3-secrets/#secrets-in-s3
   # encryptedKubeconfig: encrypted:s3!r:us-west-2!b:mybucket!f:mykubeconfig
-  # List of contexts from the kubeconfig to make available to Spinnaker
-  contexts:
-  - default
+  accounts:
+    # default:
+      # context: default
+      # Use this to limit the namespaces this kubernetes account will access.
+      # Note, if you use `namespaces`, global `omittedNameSpaces` will not be used.
+      # namespaces:
+      #   - namespace1
+      #   - namespace2
+      # Use this to limit the Spinnaker RBAC roles than can read from Kubernetes.
+      # readPermissions:
+      #   - role1
+      #   - role2
+      # Use this to limit the Spinnaker RBAC roles than can write (i.e. Deploy) to Kubernetes.
+      # writePermissions:
+      #   - role1
+      #   - role2
   # The Spinnaker account that will be used to deploy the Spinnaker cluster to.
   deploymentAccount: default
-  # Use this to limit the namespaces this kubernetes account will access.
-  # Note, if you use nameSpaces, omittedNameSpaces will not be used.
-  # Note, the casing of the nameSpace variable here.
-  # nameSpaces:
-  # - namespace1
-  # - namespace2
   omittedNameSpaces:
```

Comment:
Think we should restore some of the documentation here still, or move this closer to `accounts`.

Reply:
There was no clear documentation previously, however, I now added some. PTAL.